src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -549,6 +552,9 @@
         - name: --vnet-subnet-id
           type: string
           short-summary: The ID of a subnet in an existing VNet into which to deploy the cluster.
+        - name: --pod-subnet-id
+          type: string
+          short-summary: The ID of a subnet in an existing VNet into which to deploy assign pods in the cluster
```

Comment:
should we say that this requires all existing agentpools to have pod subnet id?

Reply:
No letting validation in RP handle that since it may change.